docs/cmd/tkn_pipelinerun.md

```diff
@@ -0,0 +1,41 @@
+## tkn pipelinerun
+
+Pipelinerun command group
+
+### Synopsis
+
+Pipelinerun command group
+
+### Aliases
+
+```
+pipelinerun, pr, pipelineruns
+```
+
+### Available Commands
+
+```
+describe    Describe a pipelinerun in a namespace
+list        Lists pipelineruns in a namespace
+logs        Show the logs of PipelineRun
+
+```
+
+### Options
+
+```
+-h, --help                help for pipelinerun
+-k, --kubeconfig string   kubectl config file (default: $HOME/.kube/config)
+-n, --namespace string    namespace to use (default: from $KUBECONFIG)
+```
+
+### Options inherited from parent commands
+
+```
+--azure-container-registry-config string   Path to the file containing Azure container registry configuration information.
+```
+
+### SEE ALSO
+
+* [tkn pipelinerun list](tkn_pipelinerun_list.md)	 - List all pipelineruns in given namespace.
```

Comment:
Outside of the CLI command it looks like PipelineRun, Task, etc... are presented as `PipelineRun`, `Task`, etc...

Reply:
We can go with `pipelinerun`, `task`, etc.. to maintain consistency.